memory/combat_manager.py

```diff
@@ -0,0 +1,150 @@
+from enum import Enum
+
+from memory.core import mem_handle
+
+
+class CombatDamageType(Enum):
```

Comment:
I assume two of the unknown ones are blunt/hammer and Sun, not sure what the rest could be.

Reply:
yep~! I just haven't taken the time to find those specific yet, but there's apparently a few.